cosign/README.md

```diff
@@ -0,0 +1,125 @@
+# Cosign Verifier
+
+## Introduction
+
+In order to provide signatures over arbitrary artifacts, Sigstore performs three major operations[1]:
+
+1. OIDC Issuance, vouch that a client is in control of an identity
+2. Associate short-lived public key certificates with these identities (from a Certificate Authority), and publish these certificates to an Identity Log.
+3. Publish a long-lived signature over an artifact (or artifact meta-data) to an Artifact Log, allowing verifiers to check its validity.
+
+The first and second operations are performed by a system called Fulcio, which functions as a Certificate Authority and transparency log for a namespace of OIDC identities (Identity Log).
+The third operation is performed by a system called Rekor, a transparency log for artifact signatures (Artifact Log).
+And cosign is the reference Sigstore client implementation to sign arbitrary artifacts. With these components, Sigstore completed trust setup, signing and verification[2].
+
+```mermaid
+flowchart TB
+  %% OIDC Flow
+  Signer["💻 Signer"] -->|① OIDC Challenge Start| OIDC["@ OIDC Provider"]
+  OIDC -->|② OIDC Response| Signer
+  
+  %% Certificate Issuance via Fulcio
+  Signer -->|③ Issuance Request| Fulcio["📜 Fulcio (Certificate Authority)"]
+  Fulcio -->|④ Signed Certificate| Signer
+  Fulcio -->|Publish to Identity Log| IdentityLog
+  
+  %% Signing & Publishing Artifacts
+  Signer -->|⑤ Sign and Publish Artifact| ArtifactLog
+  Signer -->|⑤ Sign and Publish Artifact| ArtifactRepo["📦 Artifact Repository"]
+  
+  %% Identity and Artifact Logs
+  subgraph Logs["Transparency Logs"]
+    subgraph IdentityLog["📜 Identity Log (Fulcio)"]
+      ID1["UUID: 5a88c7 SAN: user#64;github.com"]
```

Comment:
Fulcio seems do not utilize `UUID` and `SAN`.

Reply:
https://github.com/sigstore/fulcio/blob/1a642e565a7620f25f6ac3a32a430dd7168902a4/docs/oidc.md?plain=1#L55